core/types/hashing.go

```diff
@@ -67,8 +60,7 @@ func rlpHash(x interface{}) (h common.Hash) {
 // prefixedRlpHash writes the prefix into the hasher before rlp-encoding x.
 // It's used for typed transactions.
 func prefixedRlpHash(prefix byte, x interface{}) (h common.Hash) {
-	sha := hasherPool.Get().(crypto.KeccakState)
-	defer hasherPool.Put(sha)
+	sha := crypto.NewKeccakState()
```

Comment:
This function is used for hashing tx/recover sender. Originally the keccak buffer will be allocated from the pool.

We should somehow keep the syncPool, or use some alternative ways.